tests/test_arcaflow_plugin_stressng.py

```diff
@@ -14,6 +14,30 @@
 class StressNGTest(unittest.TestCase):
     @staticmethod
     def test_serialization():
+        plugin.test_object_serialization(
+            stressng_schema.StressNGParams(
```

Comment:
While you're here, consider adding test cases for `IomixStressorParams` and `SockStressorParams`.  (And, maybe the corresponding `test_functional_*()` cases, as well.)